CAMERA_TYPES

```diff
@@ -1,20 +1,11 @@
-LIF - A camera supporting a connection to LIVE_IMAGE_FULL, with N_OF_COL
-      and N_OF_ROW being the image size.
-AVG - A camera supporting LIVE_IMAGE_FULL and an on-IOC averaged AVG_IMAGE.
-      N_OF_COL and N_OF_ROW are the image size, and AVERAGER.A is the number
-      of frames to average.
-IC  - A camera supporting a connection to IMAGE_CMPX, a compressed image.
-      Control of the compression is via SHIFT.VAL (bit shift), ROI_X, ROI_Y,
-      ROI_XNP, and ROI_YNP.  Images are <= 512x512.  Actual size is given
-      by COMPRESSOR.VALE (rows) and COMPRESSOR.VALF (cols)
-GE  - A gigE camera.  The image data is bytes in ArrayData, and the dimensions
-      are in ArraySize0_RBV, ArraySize1_RBV, and ArraySize2_RBV.  If color,
-      these are 3, X, Y, and if B/W, these are X, Y, 0.  No EVR!
-MCC - Image is IMAGE, N_OF_COL, N_OF_ROW, and N_OF_BITS.  Or is this the ROI image?
-      and so ROI_X,ROI_XNP, ROI_Y, ROI_YNP are the actual size of IMAGE?
-LI  - Image is LIVE_IMAGE, size is N_OF_ROW/2, N_OF_COL/2.
-LIO - Image is LIVE_IMAGE_12B, size is N_OF_ROW/2, N_OF_COL/2.
-LIX - Image is LIVE_IMAGE_12B, size is ROI_XNP/2, ROI_YNP/2.
+AD  - An AreaDetector camera.  The image data is bytes in ArrayData,
```

Comment:
:+1: on dropping the legacy camera types as we approach 2023 and everything is AD now

Reply:
Yeah.  Sorry CXI, but you should have moved away from unixCam ages ago...